crm_compassion/model/demand_planning.py

```diff
@@ -0,0 +1,45 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Maxime Beck
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp import models, fields
+
+
+class WeeklyDemand(models.Model):
+
+    _name = 'demand.weekly.demand'
+    _description = 'Weekly Demand'
+
+    name = fields.Char(required=True)
+    demand_id = fields.Many2one(
```

Comment:
change to `demand_ids` and to a Many2Many field. We will reuse weekly demand across several plannings.